kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/kms/aws/AwsKmsCloud.java

```diff
@@ -39,7 +33,10 @@ public boolean isAvailable() {
 
     @Override
     public void deploy() {
-        // nothing to deploy
+        ExecResult execResult = Exec.exec(AWS_CMD, "configure", "set", "region", Environment.AWS_REGION);
```

Comment:
surprised we need this

Reply:
Having some tests, I realized that we need to configure the region for AWS Cloud when running pipelines. I had several failures saying that
```
RecordEncryptionST.ensureClusterHasEncryptedMessage:81 » KubeCluster Failed to run AWS Kms: [aws, kms, create-key], exit code: 255, stderr: You must specify a region. You can also configure your region by running "aws configure".
```

Reply:
That doesn't make sense.  `aws configure` is just configuring the CLI.  The objective was to eliminate use of CLI, so there ought to be no dependency on it.

Reply:
I agree but to access AWS Cloud URL we have to know on which region we are:
```
URI.create("https://kms." + getRegion() + ".amazonaws.com");
```
If we don't use CLI to get or to configure the region, how can we know it? Or can we just set the region we want on the URL?
Let me try that.

Reply:
Great, it works fine. I thought wrongly that we needed to configure the region in the CLI for that.